pkgs/games/unvanquished/default.nix

```diff
@@ -0,0 +1,139 @@
+{ lib, stdenv, writeScript, fetchzip, fetchurl, fetchgit, buildFHSUserEnv, runCommandNoCC, makeDesktopItem
+, gcc, cmake, gmp, libGL, zlib, ncurses , geoip, lua5, nettle
+, curl, SDL2, freetype, glew , openal, libopus, opusfile, libogg
+, libvorbis, libjpeg, libwebp, libpng
+, libtheora # TODO: drop once we get to 0.52
+, cacert, aria2 # to download assets
+}:
+
+let version = "0.51.2";
+    binary-deps-version = "4";
+
+    unvanquished-binary-deps = stdenv.mkDerivation rec {
+      # DISCLAIMER: this is selected binary crap from the NaCl SDK
+      pname = "unvanquished-binary-deps";
+      version = binary-deps-version;
+      src = fetchzip {
+        url = "https://dl.unvanquished.net/deps/linux64-${version}.tar.bz2";
+        sha256 = "08bpyavbh5lmyprvqqi59gnm8s1fjmlk9f1785wlv7f52d9f9z1p";
+      };
+      dontPatchELF = true;
+      preFixup = ''
+        # We are not using the autoPatchelfHook, because it would make
+        # nacl_bootstrap_helper unable to load nacl_loader:
+        # "nacl_loader: ELF file has unreasonable e_phnum=13"
+        interpreter="$(< "$NIX_CC/nix-support/dynamic-linker")"
+        for f in pnacl/bin/*; do
+          if [ -f "$f" && -x "$f" ]; then
```

Comment:
```suggestion
          if [[ -x $f ]]; then
```

Reply:
Unfortunately, this isn't quite the same thing, as there is a subdirectory here, and it technically is "executable" (it has the x flag set)